src/sysv/systemd/sssd-sudo.service.in

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=SSSD Sudo Service responder
+Documentation=man:sssd.conf(5)
+After=sssd.service
+BindsTo=sssd.service
+
+[Install]
+Also=sssd-sudo.socket
+
+[Service]
+ExecStartPre=-/bin/chown @SSSD_USER@:@SSSD_USER@ @logpath@/sssd_sudo.log
+ExecStart=@libexecdir@/sssd/sssd_sudo --debug-to-files --unprivileged-start
```

Comment:
Ouch! Yeah, I've missed that one while re-working the patches.

Shall I push a new series now or would you prefer to fix it locally and keep the code review? (I would prefer the latter).